e2e/repo-page.spec.ts

```diff
@@ -53,21 +51,15 @@ test.describe("small screen", () => {
     await page.goto("/s/open-sauced/app");
 
     await page.getByRole("button", { name: "Add to Workspace", exact: true }).click();
-    const dialog = await page.getByRole("dialog", { name: "Add repository to Workspace", exact: true });
+    const dialog = page.getByRole("dialog", { name: "Add repository to Workspace", exact: true });
     await expect(dialog).toBeVisible();
-
-    await dialog.getByRole("button", { name: "Connect with GitHub", exact: true }).click();
-    await expect(page.url()).toContain("https://github.com/login");
   });
 
   test("Adds a repository SBOM to a workspace", async ({ page }) => {
     await page.goto("/s/open-sauced/app");
 
     await page.getByRole("button", { name: "Workspace from SBOM", exact: true }).click();
-    const dialog = await page.getByRole("dialog", { name: "Add repository SBOM to Workspace", exact: true });
+    const dialog = page.getByRole("dialog", { name: "Add repository SBOM to Workspace", exact: true });
     await expect(dialog).toBeVisible();
-
-    await dialog.getByRole("button", { name: "Connect with GitHub", exact: true }).click();
```

Comment:
If we remove the remove the click, we can probably still check for the login button in the dialog, but again, need to see what's up with the click not firing consistently to check the login URL.